src/seismometer/builder/resources/classifier_bin.ipynb

```diff
@@ -309,6 +318,7 @@
    },
    "source": [
     "**Tips**: \n",
+    " - See our documentation on [feature monitoring](https://epic-open-source.github.io/seismometer/user_guide/index.html#feature-monitor) for more details.\n",
```

Comment:
Nit: Is this the voice we want references to have?  
I know 'our' is meant to feel community-driven but could be interpreted as authoritative.  Might be better to avoid with a "More details can be found here"

Reply:
finc